src/gui/plugins/modules/EntityContextMenu.qml

```diff
@@ -63,6 +63,14 @@ Item {
     id: secondMenu
     x: menu.x + menu.width
     y: menu.y + viewSubmenu.y
+    MenuItem {
```

Comment:
Can you move this menu item below `Collsions` so that they are alphabetical?

Reply:
Changed in [`1393204`](https://github.com/ignitionrobotics/ign-gazebo/pull/816/commits/139320417f89a01a1b77f7b2d0117a8d0f6990da)